### PR TITLE
add snowflakes

### LINKED ISSUE #533: Fix the wording of the rule regarding multiple accounts
The rule on multiple accounts on the website should be reworded:
>Do not abuse by using multiple accounts."

This currently implies that it's fine to have multiple accounts as long as you don't cause abuse.
Suggested to reword it to:  "Do not create more than one account", as that is the intended meaning.

### PR DESCRIPTION
**Pull request by fcaps**
closes #533

### CODE REVIEW

**Comment by fcaps** [comment]
snow

**Comment by Askaholic** [comment]
Should it turn on/off automatically based on the date? That would probably be better than requiring a developer to merge a code change and have it get deployed each time.

**Comment by fcaps** [comment]
> Should it turn on/off automatically based on the date? That would probably be better than requiring a developer to merge a code change and have it get deployed each time.

also thought about it, but then decided to keep it simple.
you can update this PR if you think this would be good to implement.

**Review by Brutus5000 — APPROVED** [review]
